src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -106,38 +102,11 @@ export class MeetingManager implements AudioVideoObserver {
     fullDeviceInfo: FullDeviceInfoType
   ) => void)[] = [];
 
-  logLevel: LogLevel = LogLevel.WARN;
-
-  postLoggerConfig: PostLogConfig | null = null;
-
   eventReporter: EventReporter;
 
-  simulcastEnabled: boolean = false;
-
-  videoDownlinkBandwidthPolicy: VideoDownlinkBandwidthPolicy | undefined;
-
-  logger: Logger | undefined;
-
   private meetingEventObserverSet = new Set<(name: EventName, attributes: EventAttributes) => void>();
 
-  constructor(config: ManagerConfig) {
-    if (config.simulcastEnabled) {
-      this.simulcastEnabled = config.simulcastEnabled;
-    }
-
-    if (config.logger) {
-      this.logger = config.logger;
-    } else {
-      this.logLevel = config.logLevel;
-      if (config.postLogConfig) {
-        this.postLoggerConfig = config.postLogConfig;
-      }
-    }
-
-    if (config.videoDownlinkBandwidthPolicy) {
-      this.videoDownlinkBandwidthPolicy = config.videoDownlinkBandwidthPolicy;
-    }
-  }
+  constructor(private config: ManagerConfig) { }
```

Comment:
Wish we could call this something better than config to avoid confusion with this.configuration.